docker-compose-federation2.yml

```diff
@@ -1,6 +1,19 @@
 version: "3.9"
 services:
 
+  gateway:
+    container_name: apollo-gateway
+    build: dockerfiles/federation2-demo/gateway
+    environment:
+      - APOLLO_OTEL_EXPORTER_TYPE=collector
+      - APOLLO_OTEL_EXPORTER_HOST=collector
+      - APOLLO_OTEL_EXPORTER_PORT=4318
+      - APOLLO_SCHEMA_CONFIG_EMBEDDED=true
+    volumes:
+      - ./dockerfiles/federation2-demo/supergraph.graphql:/etc/config/supergraph.graphql
```

Comment:
is this intentional? not sure how this ties to the pull request

Reply:
Yep. This looks like it was included accidentally.

Reply:
No it's intentional, I need it when running apollo-smith, in the fed1 version that was done like that